EssentialFeed/EssentialFeed/Image Comments/Presentation/ImageCommentsPresenter.swift

```diff
@@ -0,0 +1,25 @@
+//
+// Copyright © 2021 Essential Developer. All rights reserved.
+//
+
+import Foundation
+
+public final class ImageCommentsPresenter {
+	public static var title: String {
+		NSLocalizedString(
+			"IMAGE_COMMENTS_VIEW_TITLE",
+			tableName: "ImageComments",
+			bundle: Bundle(for: ImageCommentsPresenter.self),
+			comment: "Title for the image comments view")
+	}
+
+	public static func map(_ comments: [ImageComment], currentDate: Date = Date(), calendar: Calendar = Calendar(identifier: .gregorian), locale: Locale = Locale(identifier: "en_US_POISX")) -> ImageCommentsViewModel {
+		ImageCommentsViewModel(comments: comments.map { comment in
+			let formatter = RelativeDateTimeFormatter()
+			formatter.calendar = calendar
+			formatter.locale = locale
```

Comment:
`RelativeDateTimeFormatter` is expensive to create and configure. So it shouldn't be created within loops (like Array.map). Instead, create and configure the formatter once and reuse it in the map:

```suggestion
		let formatter = RelativeDateTimeFormatter()
		formatter.calendar = calendar
		formatter.locale = locale
		
		return ImageCommentsViewModel(comments: comments.map { comment in
			
```